src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -484,27 +486,40 @@ export default class DefaultAudioVideoController
     this.meetingSessionContext.enableSimulcast = this.enableSimulcast;
 
     if (this.enableSimulcast) {
-      const simulcastPolicy = new DefaultSimulcastUplinkPolicy(
-        this.configuration.credentials.attendeeId,
-        this.meetingSessionContext.logger
-      );
+      let simulcastPolicy = this.meetingSessionContext
+        .videoUplinkBandwidthPolicy as SimulcastUplinkPolicy;
```

Comment:
Just ignore the flag if they specified a policy. Don't overrule the customer's intent.

Reply:
We already handle that in DefaultMeetingSession.ts right? I set the flag if they pass in their own simulcast policy.